client/app/components/queries/QueryEditor/QueryEditorControls.jsx

```diff
@@ -84,6 +86,13 @@ export default function EditorControl({
           onToggle={autocompleteToggleProps.onToggle}
         />
       )}
+      {autoLimitCheckboxProps !== false && (
+        <AutoLimitCheckbox
+          available={autoLimitCheckboxProps.available}
+          checked={autoLimitCheckboxProps.checked}
+          onChange={autoLimitCheckboxProps.onChange}
+        />
```

Comment:
This one is not a must, but @kravets-levko do you see any reason not to just 
```jsx
<AutoLimitCheckbox {...autoLimitCheckboxProps} />
```

In here and in similar ones?

Reply:
No, I see no reason for not just using a spread operator